### PR TITLE
feat: add fluentd container for logging

### PR DESCRIPTION
**Pull request by alangsto**
Configuration Pull Request
---
Add a fluentd container that handles tracking logs from edxapp. When edxapp is containerized, the logging config will be configured to send tracking logs to a specified port. Fluentd is configured to listen to that port, and then output to std.out. In the future, fluentd could be configured to also output to S3.

Make sure that the following steps are done before merging:

  - [ ] A SRE team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a SRE ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/spaces/SRE/pages/28967861/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).
  - [ ] Think about how this change will affect Open edX operators.  Have you updated the wiki page for the next Open edX release?

### CODE REVIEW

File: util/jenkins/ansible-provision.sh [hunk 1]
```diff
@@ -788,5 +788,37 @@ EOF
...
+        <parse>
+            @type none
+        </parse>
+    </source>
+
+    <match **>
+        @type stdout
+    </match>
+EOF"
+    echo "docker run -d -u root -v /var/tmp/fluentd.conf:/fluentd/etc/fluentd.conf -v /edx/var/log/tracking:/edx/var/log/tracking fluent/fluentd:edge-debian -c /fluentd/etc/fluentd.conf"
```
Comment (alangsto) [inline util/jenkins/ansible-provision.sh]: have to run as root user in order to properly mount the tracking log directory. Depending on how we use fluentd as a logging driver for edxapp, I'm not sure we'll need to do the same thing

**Review by schenedx — COMMENTED** [review]
Looks pretty good to me. When do you think this is out of the draft state? Maybe after https://github.com/openedx/edx-platform/pull/31217 is merged?

**Comment by alangsto** [comment]
The platform PR should be merged, but I'd feel more comfortable merging this once we have the CMS/LMS actually running in a container. 

If we want to merge this now, it should definitely be put behind some sort of gate.

**Review by schenedx — APPROVED** [review]
👍

**Pull request merged**